### PR TITLE
Team Spotlight - SDK team

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: blog/news-and-updates/team-spotlight-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Hey devs and welcome to this edition of the IC team spotlight! In this issue, we sat down with the SDK team to talk all things IC SDK and the upcoming Cycles Ledger feature! For most IC devs, the IC SDK is at the center of their workflow, so I'm sure many of you have been looking forward to this team spotlight! With that, let's dive into the interview. 
+
+**To get started, let's get a brief overview of the team. What is the purpose of the SDK team?**
+
+*To provide tools that make building on the Internet Computer fast, easy, and fun.*
+ 
+**What is the composition of the SDK team?**
+
+*The SDK team has seven members. The engineers on the SDK team are Eric Swanson, Linwei Shang, Kyle Peacock, Severin Siffert, Marcin Nowak-Liebiediew,  and Adam Spofford. The team also includes Jessie Mongeon, a technical writer, Thomas Locher, a visiting member from the research team, and Jason Ibrahim, the team’s engineering manager.*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-sdk.mdx:16]: has everyone consented to using their full names? we want to respect people's privacy. should we change this to something more generic perhaps?

File: blog/news-and-updates/team-spotlight-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+**When can we expect the Cycles Ledger to be completed?**
+
+*The end of the year. It will be our holiday gift to you :)*
+
+**I'm sure that since the IC SDK is such a fundamental part of the developer experience, the SDK team must interact with other teams quite frequently. How does the SDK team work with other teams?**
+
+*There are a number of “sister” teams to SDK, such as Execution and Runtime. We rely on the code shipped by them (replica) and they also rely on us for developer-driven feedback that helps us make the replica (and underlying platform) better in all aspects.*
+
+*We also occasionally work with other teams when there is a cross-functional initiative going on. One example was when we developed the Service Nervous System. We worked closely with the SNS team to ensure that the tooling we provided met all of the specifications.*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-sdk.mdx:62]:
```suggestion
*We also occasionally work with other teams when there is a cross-functional initiative going on. One example was when the Service Nervous System was developed. SDK worked closely with the SNS team to ensure that the tooling provided met all of the specifications.*
```

File: blog/news-and-updates/team-spotlight-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+**The SDK team is pretty involved with the dev community on the forum and Discord server, which I'm sure has lead to features derived as a result of community feedback. What are some of the improvements made as a direct result of developer feedback?**
+
+*Lots! We built the [developer feedback board](http://dx.internetcomputer.org) a while back, and every two weeks, our internal focus group meets to go over the latest items and check-in on the progress we’ve made addressing others. One thing we’ve built as a result of the feedback we’ve received is the `dfx` version manager. Developers expressed that they want a way to switch `dfx` versions easily. We spent some time working out how best to accomplish that, and the `dfx` version manager was born. Another example is the Cycles Ledger - we heard feedback that the Cycles Wallet was a pain to use and developers wanted a better solution. We are continuously looking for ways to improve our tooling, so keep the feedback coming!*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-sdk.mdx:70]:
```suggestion
*Lots! We built the [developer feedback board](http://dx.internetcomputer.org) a while back, and every two weeks, our internal focus group meets to go over the latest items and check-in on the progress we’ve made addressing others. One thing that's come as a result of the feedback received is the `dfx` version manager. Developers expressed that they want a way to switch `dfx` versions easily. We spent some time working out how best to accomplish that, and the `dfx` version manager was born. Another example is the Cycles Ledger - we heard feedback that the Cycles Wallet was a pain to use and developers wanted a better solution. We are continuously looking for ways to improve our tooling, so keep the feedback coming!*
```

**Review by dfx-json — APPROVED** [review]

File: blog/news-and-updates/team-spotlight-sdk.mdx (hunk 1 above)
Comment (jessiemongeon1) [reply blog/news-and-updates/team-spotlight-sdk.mdx]: Removed names and made generic 👍🏻

**Pull request merged**